micropub.php

```diff
@@ -20,6 +20,23 @@
 //   curl -i -d 'code=CODE&me=SITE&client_id=indieauth&redirect_uri=https://indieauth.com/success' 'https://tokens.indieauth.com/token'
 // 4. Extract the access_token parameter from the response body.
 
+// For debugging purposes this will bypass Micropub authentication
```

Comment:
huh. did you notice that it doesn't require auth when it's running on localhost? given that, do you think still we want this?